CheckboxesValidation/Student.java

```diff
@@ -0,0 +1,57 @@
+package SpringMVCBasics;
+
+import java.util.LinkedHashMap;
+
+public class Student {
+
+	private String firstName;
+	private String lastName;
+	private String[] operatingSystems;
+	private LinkedHashMap<String, String> countryOptions;
```

Comment:
Any Specific Reason on Using LinkedHashMap ?
What happens if you use just Map ?

Reply:
Just to maintain the order of insertion I think